src/SqlVirtualMachine/SqlVirtualMachine/SqlVirtualMachine/Cmdlet/NewAzureSqlVM.cs

```diff
@@ -75,11 +75,10 @@ public class NewAzureSqlVM : AzureSqlVMUpsertCmdletBase
         /// <summary>
         /// License type of new sql virtual machine
         /// </summary>
-        [Parameter(Mandatory = true,
+        [Parameter(Mandatory = false,
```

Comment:
You need to update the help documentation as well: New-AzSqlVM.md
https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md#updating-help-after-making-cmdlet-changes